config.toml

```diff
@@ -1,8 +1,8 @@
 # The URL the site will be built for
-base_url = "https://2024.matrix.org"
+base_url = "https://2025.matrix.org"
```

Comment:
Doesnt this REQUIRE a new domain? So a ping to SRE? @thibaultamartin

Reply:
try opening it ;)

Reply:
Turns out this was already resolved